poc_example fault injection fails on GitHub-hosted ubuntu; passes in Lima

Discovered on the first CI run of PR #55 (testops harness).

## Symptom

On \`ubuntu-latest\` (GitHub-hosted runner, kernel 6.x), running the testops harness's \`poc_example\` case:

\`\`\`
--- FAIL: test_db_slow (209ms, seed=1) ---
  reason: assert_eq failed: 0 != 200

--- FAIL: test_happy_path (10208ms, seed=1) ---
  reason: tcp send failed: read: read tcp 127.0.0.1:34466->127.0.0.1:33125: i/o timeout
\`\`\`

The same seed + spec + binaries passes cleanly in Lima (Ubuntu 24.04, kernel 6.8.0, aarch64). Faultbox's setup logs look identical up to \`service_started\` — PID/MNT/USER namespaces install, seccomp filter loads, target execs. Failures appear only when the real client inside \`mock-api\` tries to reach \`mock-db\` via loopback TCP.

## Impact

Blocks most Critical-tier rows in [docs/feature-manifest.md](../blob/main/docs/feature-manifest.md) from reaching green in CI — fault injection, real-syscall temporal assertions, and the poc_example end-to-end path all depend on the seccomp sandbox working on shared runners.

## Suspected causes, in order of likelihood

1. **AppArmor / container profile:** GitHub-hosted runner restricts the network or user-namespace sandbox that faultbox sets up, making in-sandbox services unable to accept localhost TCP from the outer harness.
2. **sysctl difference:** \`/proc/sys/kernel/unprivileged_userns_clone\` or related differs, causing the USER namespace setup to produce a subtly different mapping that breaks loopback reachability.
3. **Timing:** CI runners are I/O-noisier than Lima; 500ms-delay fault plus healthcheck window may not leave enough budget for the real HTTP round-trip on \`test_db_slow\`.

## Repro

Push any branch with \`poc_example\` un-skipped in \`testops/harness.go\`:

\`\`\`
# in testops/harness.go, remove the Skip: line from the poc_example Case
git push
# Default CI workflow reproduces within 15s.
\`\`\`

## Acceptance

\`go test ./testops/... -run poc_example\` passes on \`ubuntu-latest\` GitHub-hosted runner. Likely fixes: running faultbox under sudo on CI; loosening the namespace config for shared-runner environments; documenting the required runner configuration.

--- a/docs/feature-manifest.md
+++ b/docs/feature-manifest.md
@@ -33,13 +33,13 @@ Status legend: **🟢 green** (CI signal proves it), **🟡 partial** (some mech
 
 | Feature | Tier | Mechanism | Status | Notes |
 |---|---|---|---|---|
-| `service()` binary mode (fork+exec) | 1 | testops corpus (LinuxOnly) + integration | 🟡 | Blocked on #57 for CI coverage |
+| `service()` binary mode (fork+exec) | 1 | testops corpus (poc_example, poc_demo) | 🟢 | Gated in CI on amd64 ubuntu-latest |
 | `service()` container mode (Docker) | 1 | testops corpus with pinned image digest | 🔴 | CI Docker provisioning not yet added |
-| `fault(deny)` on syscalls | 1 | testops corpus + syscall-family test | 🔴 | Blocked on #57 |
-| `fault(delay)` on syscalls | 1 | testops corpus | 🔴 | Blocked on #57 |
-| `fault(hold)` on syscalls | 1 | testops corpus | 🔴 | No corpus entry yet |
-| `assert_eventually` temporal | 1 | testops corpus (real syscall trace) | 🔴 | Matching bug tracked in #56 |
-| `assert_never` temporal | 1 | testops corpus | 🔴 | No corpus entry yet |
+| `fault(deny)` on syscalls | 1 | testops corpus (poc_example test_api_cannot_reach_db; poc_demo test_wal_fsync_failure, test_disk_full) | 🟢 | |
+| `fault(delay)` on syscalls | 1 | testops corpus (poc_example test_db_slow; poc_demo test_inventory_slow) | 🟢 | |
+| `fault(hold)` on syscalls | 1 | testops corpus | 🔴 | No corpus entry exercises hold |
+| `assert_eventually` temporal | 1 | testops corpus (poc_demo test_happy_path on openat) | 🟢 | |
+| `assert_never` temporal | 1 | testops corpus (poc_demo test_inventory_unreachable) | 🟢 | |
 | `--seed` deterministic replay | 1 | testops harness itself asserts identical traces across 5 runs | 🟢 | Already proven by corpus entries |
 | `depends_on` + healthcheck ordering | 1 | testops corpus covers transitively | 🟢 | |
 | Starlark spec language | 1 | Parser unit tests + corpus exercises | 🟢 | |
@@ -107,15 +107,17 @@ Protocol-level fault proxy rewrites wire-level responses. Critical because this 
 
 ---
 
-## Summary counts (provisional, pre-confirmation)
+## Summary counts
 
-- Critical (Tier 1): 17 rows, **~12% green**.
+- Critical (Tier 1): 17 rows, **~47% green**.
 - Supported (Tier 2): 22 rows, **~55% green**.
 - Experimental (Tier 3): 8 rows, **~0% green** — expected; these are checklist-gated.
 
-The Critical gap is the story: core-workflow coverage is underwater.
-Issue #57 (shared-runner seccomp) is the single highest unlock —
-resolving it turns ~8 Critical rows from 🔴 to 🟢 at once.
+PR #64 (proxy teardown, closes #57 + #61) + PR #62 (openat matching,
+closes #56) unlocked 5 Critical rows at once by turning on real
+seccomp fault-injection coverage in CI on amd64 ubuntu-latest.
+Next largest gap: Docker-backed cases (#poc_demo_container,
+#poc_kafka_rfc014), which require CI Docker provisioning.
 
 ---
 

--- a/internal/star/results.go
+++ b/internal/star/results.go
@@ -493,6 +493,17 @@ func NormalizeTrace(result *SuiteResult) string {
 					continue
 				}
 
+				// Skip allowed system-path accesses regardless of the
+				// specific path. Go's runtime probes cgroup/proc/sys
+				// paths at startup that vary by arch and host (e.g.
+				// /sys/fs/cgroup/.../cpu.max exists on GitHub-hosted
+				// runners but not in Lima). These are never
+				// application behavior; including them in the
+				// normalized trace makes goldens non-portable.
+				if decision == "allow (system path)" {
+					continue
+				}
+
 				// Normalize non-deterministic paths:
 				// socket:[12345] → socket (inode numbers change between runs)
 				// pipe:[12345] → pipe

--- a/testops/goldens/poc_demo.norm
+++ b/testops/goldens/poc_demo.norm
@@ -1,0 +1,104 @@
+=== test_disk_full ===
+--- inventory ---
+service_started
+write allow pipe
+service_ready
+fault_applied
+write deny(no space left on device) socket
+fault_removed
+--- orders ---
+service_started
+service_ready
+--- test ---
+step_send orders
+step_recv orders
+=== test_flaky_network ===
+--- inventory ---
+service_started
+write allow pipe
+service_ready
+write allow socket
+openat allow /tmp/inventory.wal
+write allow /tmp/inventory.wal
+fsync allow /tmp/inventory.wal
+write allow socket
+--- orders ---
+service_started
+service_ready
+fault_applied
+fault_removed
+--- test ---
+step_send orders
+step_recv orders
+=== test_happy_path ===
+--- inventory ---
+service_started
+write allow pipe
+service_ready
+write allow socket
+write allow socket
+openat trace /tmp/inventory.wal
+write allow /tmp/inventory.wal
+fsync allow /tmp/inventory.wal
+write allow socket
+write allow socket
+--- orders ---
+service_started
+service_ready
+--- test ---
+step_send orders
+step_recv orders
+step_send orders
+step_recv orders
+step_send orders
+step_recv orders
+=== test_inventory_slow ===
+--- inventory ---
+service_started
+write allow pipe
+service_ready
+fault_applied
+write delay(500ms) socket
+openat allow /tmp/inventory.wal
+write delay(500ms) /tmp/inventory.wal
+fsync allow /tmp/inventory.wal
+write delay(500ms) socket
+fault_removed
+--- orders ---
+service_started
+service_ready
+--- test ---
+step_send orders
+step_recv orders
+=== test_inventory_unreachable ===
+--- inventory ---
+service_started
+write allow pipe
+service_ready
+--- orders ---
+service_started
+service_ready
+fault_applied
+connect deny(connection refused)
+fault_removed
+--- test ---
+step_send orders
+step_recv orders
+=== test_wal_fsync_failure ===
+--- inventory ---
+service_started
+write allow pipe
+service_ready
+fault_applied
+write allow socket
+openat allow /tmp/inventory.wal
+write allow /tmp/inventory.wal
+fsync deny(input/output error) /tmp/inventory.wal
+write allow socket
+fault_removed
+--- orders ---
+service_started
+service_ready
+--- test ---
+step_send orders
+step_recv orders

--- a/testops/harness.go
+++ b/testops/harness.go
@@ -103,7 +103,6 @@ var Cases = []Case{
 		Seed:      1,
 		Timeout:   90 * time.Second,
 		LinuxOnly: true,
-		Skip:      "test_happy_path assert_eventually on /tmp/inventory.wal openat fails deterministically in Lima — see FINDINGS #2",
 	},
 	{
 		Name:      "poc_demo_container",

